src/reference/glossary.rst

```diff
@@ -0,0 +1,75 @@
+========
+Glossary
+========
+
+.. glossary::
+
+   Augur
+
+      A command-line application used for phylogenetic analysis. :doc:`Documentation<augur:index>`
+
+   Auspice
+
+      A web application used for phylogenetic visualization and analysis. :doc:`Documentation<auspice:index>`
+
+   workflow
+      also *pathogen workflow*, *pathogen analysis*, *Nextstrain workflow*
+
+      A reproducible process comprised of one or more :term:`builds<build>` producing :term:`datasets<dataset>`, which can be visualized by :term:`Auspice`. Implementation varies per workflow, but generally they are run by workflow managers such as Snakemake.
+
+      Our core workflows can be divided into two types:
+
+      1. Single-build workflow (e.g. Zika workflow): one build producing one dataset.
+      2. Multi-build workflow (e.g. SARS-CoV-2 workflow): multiple builds producing multiple datasets.
+
+      .. note::
+
+         The individual builds in a multi-build workflow are also "workflows" in the definition of workflow managers like Snakemake.
+
+   workflow repository
+      also *pathogen workflow repository*
+
+      A version-controlled folder containing all files necessary to run a :term:`workflow`.
+
+   build
+      also *Nextstrain build*
+
+      *(noun)* A sequence of commands, parameters and input files which work together to reproducibly execute bioinformatic analyses and generate a :term:`dataset` for visualization with :term:`Auspice`.
+
+   build (verb)
+
+      A general term for running a :term:`workflow` (e.g. ``nextstrain build``).
+
+   build step
+
+      A modular instruction of a :term:`build` which can be run standalone (e.g. ``augur filter``), often with clear input and output files.
+
+   dataset
+      A collection of :term:`JSONs` produced by a :term:`build`. It is also the shared file prefix of the JSONs. For example ``flu/seasonal/h3n2/ha/2y`` identifies a dataset which corresponds to the files
+      :
+
+      - ``flu_seasonal_h3n2_ha_2y_meta.json``
+      - ``flu_seasonal_h3n2_ha_2y_tree.json``
+      - ``flu_seasonal_h3n2_ha_2y_tip-frequencies.json``
+
+      Some :term:`workflows<workflow>` produce a single, synonymous dataset, like Zika. Others, like seasonal flu, produce many datasets.
+
+   JSONs
+      Special ``.json`` files produced and consumed by :term:`Augur` and visualized by :term:`Auspice`. These files make up a :term:`dataset`.
```

Comment:
Minor point, but the (JSON) files visualised by Auspice are not the files consumed by Augur, although I'll note that some scripts may use them.

We have used the term "Auspice JSONs" to refer to the "dataset" in this glossary.

We also use the term "Node data JSONs" which are the JSONs consumed by augur (but not auspice).

Reply:
Removed the reference to consumption by Augur: 8e509bc35bd0bc4f69f34db76e04bb36ddb77a70

Added "Auspice JSONs" as an alternate name to "dataset": 8275fe2aec16d630a57e1a9f8f23e6184d78cefd

I don't see "Node data JSONs" referenced anywhere in the tutorials, so I'll make a note to include this one later.